src/components/utils/file.js

```diff
@@ -1,4 +1,4 @@
-import loadImage from 'blueimp-load-image/js/load-image'
+import loadImage from 'blueimp-load-image/js'
```

Comment:
no we only need `load-image` at the current version. If we update to 2.12.0 we would also need `image-scale`. We do not use any of the others.